fsw/src/generic_torquer_device.c

```diff
@@ -1,598 +0,0 @@
-/*******************************************************************************
```

Comment:
Since this file exists I think that maybe this app was generated using the older version of the template code. Should compare to latest and see.

Reply:
That file does not appear to exist on my version of the code; I believe I deleted all the 'device' files.